supernode/daemon/mgr/gc/gc_peer.go

```diff
@@ -76,7 +75,8 @@ func (gcm *Manager) gcPeer(ctx context.Context, peerID string) bool {
 	}(&wg)
 
 	wg.Wait()
-	return true
+	gcm.metrics.gcPeerCount.WithLabelValues().Inc()
+	gcm.metrics.gcPeerDuration.WithLabelValues().Observe(time.Since(startTime).Seconds())
```

Comment:
I think the metric you defined has a label "peer", so you should also add label in this metric here.